swift/Picture.swift

```diff
@@ -0,0 +1,130 @@
+import CoreGraphics
+
+import thorvg
+
+/// A Swift wrapper for ThorVG's Picture, facilitating picture manipulation.
+class Picture {
+    /// Pointer to the underlying ThorVG picture object.
+    let pointer: OpaquePointer
+
+    /// Initializes a new Picture instance with an empty ThorVG picture object.
+    convenience init() {
+        let pointer: OpaquePointer = tvg_picture_new()
+        self.init(pointer: pointer)
+    }
+
+    /// Initializes a Picture instance from an existing ThorVG picture pointer.
+    init(pointer: OpaquePointer) {
+        self.pointer = pointer
+    }
+
+    /// Loads a picture from a given file path.
+    func load(fromPath path: String) throws {
+        guard tvg_picture_load(pointer, path) == TVG_RESULT_SUCCESS else {
+            throw ThorVGError.failedToLoadFromPath
+        }
+    }
+
+    /// Loads a picture from a given string, typically containing SVG or Lottie data.
+    ///
+    /// Use the `mimeType` to indicate the data format to ThorVG for correct parsing and rendering.
+    func load(fromString string: String, mimeType: String) throws {
+        guard let cString = string.cString(using: .utf8),
+              tvg_picture_load_data(pointer, cString, UInt32(cString.count), mimeType, false) == TVG_RESULT_SUCCESS
+        else {
+            throw ThorVGError.failedToLoadFromString
+        }
+    }
+
+    /// Retrieves the size of the picture.
+    func getSize() -> CGSize {
+        var width: Float = 0
+        var height: Float = 0
+        tvg_picture_get_size(pointer, &width, &height)
+        return CGSize(width: Double(width), height: Double(height))
+    }
+
+    /// Sets the size of the picture.
+    func setSize(_ size: CGSize) {
```

Comment:
> tvg_picture_set_size: Resizes the picture content to the given width and height.

I was looking at the crop function and realised this may be better called `resize`? if so maybe we can add some documentation here as well?